typescript/infra/scripts/generate-eclipse-usdt-warp-route-config.ts

```diff
@@ -0,0 +1,23 @@
+import { writeFileSync } from 'fs';
+import { stringify as yamlStringify } from 'yaml';
+
+import { WarpRouteDeployConfigSchema } from '@hyperlane-xyz/sdk';
+
+import { getEclipseEthereumUSDTWarpConfig } from '../config/environments/mainnet3/warp/configGetters/getEclipseEthereumUSDTWarpConfig.js';
+
+async function main() {
```

Comment:
instead of needing to create a new script for each warp route, can we just have a single script that takes in the warp ID / target file?

imo not blocking to finish out this deploy, but now that we're in high gear with warp deploys I think we should try our best to make the footprint smaller for warp routes

Reply:
will add this to my list of things to improve

Reply:
created https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/4808